package.json

```diff
@@ -13,6 +13,7 @@
     "path": "node_modules/cz-conventional-changelog/"
   },
   "dependencies": {
+    "@bahmutov/parse-github-repo-url": "^0.1.0",
```

Comment:
The idea was that library was already used, just not maintained, so it was missing a few cases. I just kept adding use cases, so it is pretty simple replacement.